src/nrniv/partrans.cpp

```diff
@@ -1071,211 +1066,134 @@ extern size_t nrnbbcore_gap_write(const char* path, int* group_ids);
   Cleaned up the BBCoreGapInfo (and gap_ml).
 
   So a simple factoring of the verify and create portions suffices
-  for both files and direct. Note that direct calls back for each thread
-  in index order. So for, tid 0, verify and create will be accomplished,
-  and for tid nthread-1, cleanup of BBCoreGapInfo will take place (the
-  contents of each thread are passed back and cleaned up in coreneuron
-  after use there.
+  for both files and direct. Note that direct call gets a pointer to
+  SetupTransferInfo* gi.
+  To cleanup gi, CoreNEURON should callback with an ngroup of -1.
```

Comment:
You're right. I opportunistically reused the existing callback to avoid the trouble of making a new one. Will add a separate
callback. Have to fix the broken file transfer (checkpoint problems) anyway.